requirements/docker.txt

```diff
@@ -404,7 +405,7 @@ requests-toolbelt==1.0.0
     # via -r requirements/pip.txt
 rest-framework-generic-relations==2.2.0
     # via -r requirements/pip.txt
-rich==13.9.4
+rich==14.0.0
```

Comment:
Maybe this one as well. However, I think we only use it inside Docker for nicer colors.